libr/bin/p/bin_dex.c

```diff
@@ -1551,9 +1565,9 @@ static void parse_class(RBinFile *bf, RBinDexClass *c, int class_index, int *met
 			if (t > 0 && t < dex->header.types_size ) {
 				int tid = dex->types[t].descriptor_id;
 				if (dexdump) {
-					char *cn = getstr (dex, tid);
+					const char *cn = getstr (dex, tid);
 					rbin->cb_printf ("    #%d              : '%s'\n", z, cn);
-					free (cn);
+					// free (cn);
```

Comment:
Same here, why not to remove free?

Reply:
same as above

Reply:
getstr returns const char*, not owned, so not leaking